apps/mull-ui/src/app/pages/messages/event-messages/event-message-list.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../../components';
+import EventBullet from '../../../components/event-bullet/event-bullet';
+import './event-message-list.scss';
+
+/* eslint-disable-next-line */
+export interface EventMessageListProps {}
+
+/*TODO: Remove this and replace with query of event message channels*/
+
+const messageChannels = [
```

Comment:
is it not better to use seeders instead here?

Reply:
This is going to be replaced with a query for event channels just like for #243

Reply:
I mean are we not going to query all the events the user is participating in? can't we do that right now?